scrapers/VrxDB.yml

```diff
@@ -0,0 +1,27 @@
+name: VrxDB
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - vrxdb.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="elementor-widget-container"]//@title
+      Date:
+        selector: //span[contains(text(),"Released:")]/following-sibling::text()
+        postProcess:
+          - parseDate: January 2, 2006
+      Performers:
+        Name: //span[contains(text(),"Starring:")]/following-sibling::a
+      Tags:
+        Name: //div[@class="jet-listing jet-listing-dynamic-terms"]/a
+      Details:
+        selector: //div[@class="jet-listing-dynamic-field__content"]/text()
```

Comment:
this gets a lot of texts of which the first should be ok
maybe go all out and use something like 
`      Details: //div[@class="elementor-element elementor-element-520671f1 elementor-widget elementor-widget-jet-listing-dynamic-field"]` ?